no_log.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- hosts: all
+  gather_facts: false
+  tasks:
+    - name: task with stdout
+      shell: echo {{ secret_value }}
+      no_log: True
+
+    - name: task with stderr
+      shell: cat {{ secret_value }}
+      no_log: True
+      ignore_errors: yes
```

Comment:
@jlaska Because there's no file "foo," which is the value for `secret_value` that we pass in.